types/ember__engine/index.d.ts

```diff
@@ -44,6 +45,9 @@ export default class Engine extends EmberObject.extend(RegistryProxyMixin) {
     buildInstance(options?: object): EngineInstance;
 }
 
+// tslint:disable-next-line:no-empty-interface -- used for declaration merge
+export default interface Engine extends RegistryProxyMixin {}
```

Comment:
does this work OK *after* the class? I remember some merges have weird rules about what order they appear, but I think it might be value-value merges, not type-type merges like this

Reply:
From what we've seen, `class`-`interface` merges work correctly in either order. 🤔 I’ll keep an eye out for the other, though; that’s good to be aware of.

Reply:
Should we keep it consistent, though? 🙂